pallets/summary/src/lib.rs

```diff
@@ -1431,9 +1420,14 @@ impl<AccountId> Default for RootData<AccountId> {
         }
     }
 }
-
 impl<T: Config> OnBridgePublisherResult for Pallet<T> {
-    fn process_result(_tx_id: u32, _succeeded: bool) -> DispatchResult {
+    fn process_result(tx_id: u32, succeeded: bool) -> DispatchResult {
+        if succeeded {
+            log::info!("✅  Transaction with ID {} was successfully published to Ethereum.", tx_id);
```

Comment:
If succeeded == true, we should set `is_finalised` on `RootData` to true. Otherwise it should be false.